### ISSUE TITLE
How to handle empty cells in ExcelSource?

### ISSUE THREAD

**Issue by simhue** [roadrunnerlenny/etlbox#22]
Hello,

I'm currently writing an ETL-Pipeline to transform Excel-Data to JSON by using ETLBox. The Excel-sheet has empty cells and everytime I'm executing the pipeline I encounter a NullReferenceException. 
```
Unhandled Exception: ALE.ETLBox.ETLBoxException: Error during reading data from excel file - see inner exception for details. ---> System.AggregateException: One or more errors occurred. (Object reference not set to an instance of an object.) ---> System.NullReferenceException: Object reference not set to an instance of an object.
   at ALE.ETLBox.DataFlow.ExcelSource`1.ParseDataRow(TypeInfo typeInfo)
   at ALE.ETLBox.DataFlow.ExcelSource`1.ReadAll()
   --- End of inner exception stack trace ---
   at System.Threading.Tasks.Task.Wait(Int32 millisecondsTimeout, CancellationToken cancellationToken)
   at ALE.ETLBox.DataFlow.ExcelSource`1.ExecuteAsync()
   --- End of inner exception stack trace ---
   at ALE.ETLBox.DataFlow.ExcelSource`1.ExecuteAsync()
   at Import.Program.Main(String[] args) 
```
The pipeline works fine, when I'm reading rows without empty cells or entering a blank into the empty cells. Am I missing something?

**Comment by roadrunnerlenny** [comment]
Hi Simon, 
thanks for using ETLBox and filing a bug report. 
You didn't miss here something, I was able to reproduce the issue within my tests - it was a bug in the implementation. I fixed it with version 1.4.1 which I just uploaded to nuget & github. Please make sure that you reference this version, and the ExcelSource should work as expected. 
Let me know if you have any other issues!

**Comment by simhue** [comment]
Awesome. Thank you for the quick respond and fix!
